managing-os/templates/archlinux.md

```diff
@@ -16,27 +16,647 @@ be considered experimental as Qubes developers team use mainly Fedora-based VMs
+
+*   **Note:** You can run a single command to build all the Qubes components or you can run them each individually.
+     Both ways below:
+
+    *   Single command to build all Qubes components together: (this command can take a long time to process depending of your pc proccessing power)
+    
+        *   **$ make qubes-vm**
+        <br>
+        <br>
+<a href="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/makequbes-vm_zpspkobb3gc.png" target="_blank"><img src="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/makequbes-vm_zpspkobb3gc.png"/></a>
+        <br>
+        <br>
+        
+        
+            *   These are the indivual component 'make' commands:
+    
+                *   **$ make vmm-xen-vm**
+          
+                *   **$ make core-vchan-xen-vm**
+           
+                *   **$ make core-qubesdb-vm**
+           
+                *   **$ make linux-utils-vm**
+           
+                *   **$ make core-agent-linux-vm**
+           
+                *   **$ make gui-common-vm**
+           
+                *   **$ make gui-agent-linux-vm**
+<br>
+<br>
+
+
+<big>**10:   Make the actual Archlinux template:**</big>
+
+*   **$ make template**
+
+<br>
+<br>
+<a href="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/make_template_zpstwm18q78.png" target="_blank"><img src="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/make_template_zpstwm18q78.png"/></a>
+<br>
+<br>
+<br>
+
+
+<big>**11:   Transfer Template into Dom0**</big>
+
+*   You need to ensure these two files are in the '**noarch**' directory 
+
+    *   **$ cd /home/user/qubes-builder/qubes-src/linux-template-builder/rpm/**
+
+    *   **$ ls**   *(confirm the below two files are there)*
+
+        *   **install-templates.sh**   (script to install template in dom0)
+
+    *   **$ cd noarch**
+
+    *   **$ ls**
+
+        *   **qubes-template-archlinux-X.X.X-XXXXXXXXXXXX.noarch.rpm**  (this is the template package 'X' replaces version and build digits)
+
+
+<br>
+<br>
+<a href="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/check-template-script_zps3sknpp3r.png" target="_blank"><img src="http://i256.photobucket.com/albums/hh165/tim_welter/Qubes%20OS/check-template-script_zps3sknpp3r.png"/></a>
+<br>
+<br>
+
+*   **Transfer the two files into Dom0**
+  *Note: as there is not a typical file transfer method for Dom0, for security reasons, this less than simple transfer function has to be used*
+
+    *   Swtich to Domo and open a terminal window.
+
+    **Note:** Take care when entering these cmd strings.  They are very long and have a number of characters that are easy to mix '**-**' vs '**.**' '**<u>T</u>emplates** (correct) vs **<u>t</u>emplates** (wrong) or **Template_**'(also wrong)
+
+
+        *   **$ cd /**
+
+        *   **$ sudo qvm-run --pass-io development 'cat /home/user/qubes-builder/qubes-src/linux-template-builder/rpm/install-templates.sh' > install-templates.sh**
```

Comment:
removed the rpm transfer line 342 and corresponding image-25 line 351 

will show on next merge